src/components/layout/Accordion/Accordion.scss

```diff
@@ -0,0 +1,3 @@
+@import '../../../styles/_partials/index';
+
+.Accordion {}
```

Comment:
Thoughts on removing this file?

Reply:
Yeah, I wrestled with this. I kept on thinking we might get some design feedback on how it scrolls and such, but that never came. I'll remove this. Thanks!